app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from __future__ import annotations
```

Comment:
Unnecessary import

Reply:
Thank you for the remarks. Updated already.